### ISSUE TITLE
Jigzi Player - HopTV ג: Voiceover should be activated when tapping an item, not only while dragging it

### ISSUE THREAD

**Issue by chayaml** [ji-devs/ji-cloud#2378]
- Clicking on an item should activate the voiceover.

For instance, the user must match sounds to the correct image in a HopTV game. JiTap allows users to click on letters to hear their sound and then drag them to the right spot.
In Jigzi, the user will hear an error sound if they click/ tap on the letters and the sound can only be heard when they drag the image around the screen. 

![Screen Shot Feb 6](https://user-images.githubusercontent.com/78186896/152679795-1fe459ff-6059-466f-a770-a8f8522c3c9d.png)

**Jigzi (p14)**
https://jigzi.org/admin/curation/785a8092-69d1-11ec-b7db-4f8face28521

**JiTap (p14)**
https://jitap.net/activities/g90y/play/

This applies to many games.
